x-pack/plugins/synthetics/server/synthetics_service/synthetics_service.ts

```diff
@@ -434,17 +434,32 @@ export class SyntheticsService {
 
     const start = performance.now();
 
-    const monitors: Array<SavedObject<SyntheticsMonitorWithSecrets>> = await Promise.all(
-      encryptedMonitors.map((monitor) =>
-        encryptedClient.getDecryptedAsInternalUser<SyntheticsMonitorWithSecrets>(
-          syntheticsMonitor.name,
-          monitor.id,
-          {
-            namespace: monitor.namespaces?.[0],
+    const monitors: Array<SavedObject<SyntheticsMonitorWithSecrets>> = (
+      await Promise.all(
+        encryptedMonitors.map((monitor) => {
+          try {
+            return encryptedClient.getDecryptedAsInternalUser<SyntheticsMonitorWithSecrets>(
+              syntheticsMonitor.name,
+              monitor.id,
+              {
+                namespace: monitor.namespaces?.[0],
+              }
+            );
```

Comment:
```suggestion
            return await encryptedClient.getDecryptedAsInternalUser<SyntheticsMonitorWithSecrets>(
              syntheticsMonitor.name,
              monitor.id,
              {
                namespace: monitor.namespaces?.[0],
              }
            );
```

without await the exception thrown by `getDecryptedAsInternalUser` won't be caught by the catch block